cpp/src/arrow/python/python_to_arrow.cc

```diff
@@ -708,6 +709,26 @@ Status SerializeObject(PyObject* context, PyObject* sequence, SerializedPyObject
   return Status::OK();
 }
 
+Status SerializeTensor(std::shared_ptr<Tensor> tensor, SerializedPyObject* out) {
```

Comment:
is this code path guaranteeing 64-byte alignment of the data buffers?

Reply:
Yeah, it uses the same code as for normal tensor serialization so will be 64-byte aligned.